cli_tools/gce_ovf_export/exporter/ovf/envelope.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright (c) 2015 VMware, Inc. All Rights Reserved.
```

Comment:
do we directly make copies of these VMware files?

Reply:
Yes, unfortunately, govmomi OVF package is designed for decoding OVF descriptors, not encoding them, so I had to make a copy and make modifications to make it suitable to encoding.